### PR TITLE
add `Levi` wiki article

### PR DESCRIPTION
**Pull request by BlueChinchompa**
I imagine Levi would want to add more to this so this is marked wip.

### CODE REVIEW

File: wiki/People/Levi/en.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Levi
+
+Levi, also known as Lekuru, is the creator and primary developer of the Titanic! project.
```
Comment (iSlodinxOsu) [inline wiki/People/Levi/en.md:3]:
```suggestion
Levi, also known as Lekuru, is the creator and primary developer of the Titanic! project.
```

Comment (iSlodinxOsu) [reply wiki/People/Levi/en.md:3]: always the space at the end... why

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/People/Levi/en.md (hunk 1 above)
Comment (BlueChinchompa) [reply wiki/People/Levi/en.md:3]: You just made me learn a habbit I didn't knew I ever have.

File: wiki/People/Levi/en.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+
```
Comment (iSlodinxOsu) [inline wiki/People/Levi/en.md:6]: why 3 blanked lines or sum

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
first of all, why you added 3 empty lines

File: wiki/People/Levi/en.md (hunk 2 above)
Comment (iSlodinxOsu) [reply wiki/People/Levi/en.md:6]: perhaps delete one or two blanked lines?

File: wiki/People/Levi/en.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Levi
+
+Levi, also known as Lekuru, is the creator and primary developer of the Titanic! project.
```
Comment (iSlodinxOsu) [inline wiki/People/Levi/en.md:3]:
```suggestion
**Levi** (also known as Lekuru) is the creator and primary [[developer]] of the Titanic! project.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Comment by Lekuruu** [comment]
Maybe I could write something from my perspective here someday, and how Titanic got to be, etc.

**Review by Lekuruu — APPROVED** [review]
It should work as a base for now

**Review by iSlodinxOsu — APPROVED** [review]

**Pull request merged**